dictionaries/gdscript/cspell-tools.config.yaml

```
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/streetsidesoftware/cspell/main/packages/cspell-tools/cspell-tools.config.schema.json
+
+targets:
+  - name: 'gdscript'
+    sources:
+      - filename: 'src/gdscript.txt'
```

```suggestion
      - filename: 'src/gdscript-words/gdscript.txt'
```

No need to copy the generated words.